migrations/20181203124254-create-users.js

```diff
@@ -36,6 +36,11 @@ export default {
       type: Sequelize.STRING,
       allowNull: true,
     },
+    passwordReset: {
```

Comment:
`passwordReset` does not describe the state well enough. This is a state and should be named as such. it describes a state when the user is in the process of updating their password, right?
Something like, `resettingPassword: true`. It's a verb and should describe it as such.